### DISCUSSION TITLE
API Quota Indicator

### DISCUSSION THREAD

**Discussion by blahguy88** [ThioJoe/YT-Spammer-Purge#960] [Ideas]
I think the Spammer Purge should have a built-in indicator showing how many queries a user has used per day out of 10,000 or so. That way, the user can make more informed decisions when choosing videos to scan and spam comments to report, delete, and/or log.

**Comment by Firecul** [comment]
ThioJoe gave an answer to a similar question I asked that might be relevant.
https://github.com/ThioJoe/YT-Spammer-Purge/discussions/688#discussioncomment-2210773
